base_geoengine/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Geospatial support for Odoo
+===========================
+
+Geospatial support based on PostGIS add the ability of server to server 
+geojson to do geo CRUD and view definition.
+
+Installation
+============
+
+To install this module, you need to have `PostGIS <http://postgis.net/>`_ installed. 
+
+On Ubuntu::
+
+  sudo apt-get install postgis
+
+The module also requires two additional python libs:
+
+* `Shapely <http://pypi.python.org/pypi/Shapely>`_
+
+* `geojson <http://pypi.python.org/pypi/geojson>`_
+
+Credits
+=======
+
+Contributors
+------------
+
+* Nicolas Bessi <nicolas.bessi@camptocamp.com>
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Laurent Mignon <laurent.mignon@acsone.eu>
+* ...
```

Comment:
@nbessi it's fixed in  https://github.com/acsone/geospatial/commit/e3f05480740ecceba1b81f8d3923a6add912c63d